src/android/WebViewService.java

```diff
@@ -126,29 +126,33 @@ public void workDone() {
     }
 
     public void loadUrl() {
-        appView = makeWebView();
-        if (!appView.isInitialized()) {
-            appView.init(cordovaInterface, pluginEntries, preferences);
-        }
-        cordovaInterface.onCordovaInit(appView.getPluginManager());
-
-        WebView wv = (WebView)appView.getView();
+        try {
+            appView = makeWebView();
+            if (!appView.isInitialized()) {
+                appView.init(cordovaInterface, pluginEntries, preferences);
+            }
+            cordovaInterface.onCordovaInit(appView.getPluginManager());
 
-        int webViewId = 100
-        wv.setId(webViewId);
-        wv.setLayoutParams(new LinearLayout.LayoutParams(
-                    android.view.ViewGroup.LayoutParams.MATCH_PARENT,
-                    android.view.ViewGroup.LayoutParams.MATCH_PARENT
-        ));
+            WebView wv = (WebView)appView.getView();
 
-        //wv.getSettings().setJavaScriptEnabled(true);
+            int webViewId = 100
+            wv.setId(webViewId);
+            wv.setLayoutParams(new LinearLayout.LayoutParams(
+                        android.view.ViewGroup.LayoutParams.MATCH_PARENT,
+                        android.view.ViewGroup.LayoutParams.MATCH_PARENT
+            ));
 
+            //wv.getSettings().setJavaScriptEnabled(true);
 
-        wv.addJavascriptInterface(new JsObject(), "service");
-        //// wv.loadUrl("file:///android_asset/www/backgroundservice.html");
 
-        appView.loadUrlIntoView("file:///android_asset/www/index.html?backgroundservice=true", true);
+            wv.addJavascriptInterface(new JsObject(), "service");
+            //// wv.loadUrl("file:///android_asset/www/backgroundservice.html");
 
+            appView.loadUrlIntoView("file:///android_asset/www/index.html?backgroundservice=true", true);
+        }
+        catch (RuntimeException e) {
+            
+        }
```

Comment:
You may add some code in this block:
- free ressources (for the garbage collector)
- send a small specific exception that does not warn the user but log in Play Store or Sentry
- close threads, streams, etc.

Reply:
I moved the try catch in the calling code (the function wa sonly called once anyway). There's a `onDestroy` hook that is called automatically and frees the ressources, so I don't think there's anything to do. I aded a log, but I don't think it will show up in Google Play…